### ISSUE TITLE
grant permissions for individual sections

### ISSUE THREAD

**Issue by mrbaseman** [WBCE/WBCE_CMS#439]
like for the whole page it may be useful to allow/disallow changing an individual section to specific groups. The default could be "inherit from page permissions" but with the possibility to add other groups or exclude groups that have access to the rest of the page.

**Comment by WebDesignWorx** [comment]
Totally agree.
Visibility States like: visible, hidden, deleted etc. should also be taken in consideration.
I could think of a functionality, at least, where deleted sections are not deleted immediately but just like deleted pages behave when PAGE_TRASH is set to 'inline'.

**Comment by mrbaseman** [comment]
Yes, a section trash could be useful, too, depending on how careful users click around in the backend.

About the permissions: Sections are already shown or hidden depending on the module permissions. Maybe we just have to add a new column (similar to module permissions) in the section table.

**Comment by instantflorian** [comment]
I generally agree, specially with the idea of a section trash. 
But please keep in mind that the backend should stay clear and easy usable, so users should not be overwhelmed with too much input fields and checkboxes, especially in single-user-environements, where no per-user-management for sections is needed. 
So maybe there either should be a new setting in the general options added where it can be selected if extended section sections should be displayed or not, or at least it should give a switch for simple/advanced view on the "manage sections" page.
